packages/utils/telemetry-utils/src/errorLogging.ts

```diff
@@ -285,9 +291,13 @@ function getValidTelemetryProps(obj: any, keysToOmit: Set<string>): ITelemetryPr
             default: {
                 if (isTaggedTelemetryPropertyValue(val)) {
                     props[key] = val;
+                } else if (isSimpleArray(val)) {
+                    // we will log array object of primitives
+                    props[key] = JSON.stringify(val);
                 } else {
                     // We don't support logging arbitrary objects
                     props[key] = "REDACTED (arbitrary object)";
```

Comment:
after talking with Mark, we think it's ok to keep this line. since we now allow logging arrays, for other situations we can still see this in telemetry